### PR TITLE
DOC: organize stringmethods in what's new

### PR DESCRIPTION
**Pull request by sinhrks**
Added summarized section to the release note based on the discussion in  #9386

### CODE REVIEW

**Comment by jorisvandenbossche** [comment]
@sinhrks Thanks! But, you will have to rebase (and probably solve some conflicts after my PR that reorganized the whatsnew file a bit). But you can put the section under "New features" (eg just before 'other enhancements')

**Comment by sinhrks** [comment]
Sure, rebased.

File: doc/source/whatsnew/v0.16.0.txt [hunk 1]
```diff
@@ -120,6 +121,44 @@ from a ``scipy.sparse.coo_matrix``:
    ss = SparseSeries.from_coo(A)
    ss
 
+.. _whatsnew_0160.enhancements.string:
+
+String Methods Enhancements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (jorisvandenbossche) [inline doc/source/whatsnew/v0.16.0.txt]: Can you make this the same level as the one above (with `^^` I think)

File: doc/source/whatsnew/v0.16.0.txt [hunk 2]
```diff
@@ -120,6 +121,44 @@ from a ``scipy.sparse.coo_matrix``:
...
+
+- Following methods are accesible via ``.str`` accessor to apply the function to each values (:issue:`9282`, :issue:`9352`, :issue:`9386`, :issue:`9387`, :issue:`9439`)
```
Comment (jorisvandenbossche) [inline doc/source/whatsnew/v0.16.0.txt]: Can you add something like 'following *new* methods' or 'following methods are *additionally* accessible' to make it clear that these are new methods in `str`

also accesible  -> accessible

Comment (jorisvandenbossche) [inline doc/source/whatsnew/v0.16.0.txt]: maybe also add that it is to make it more consistent with standard methods on strings

File: doc/source/whatsnew/v0.16.0.txt [hunk 3]
```diff
@@ -120,6 +121,44 @@ from a ``scipy.sparse.coo_matrix``:
...
+``find()``     ``rfind()``    ``ljust()``    ``rjust()``        ``zfill()``
+=============  =============  =============  ===============    ===============
+
+.. ipython:: python
+
+   s = Series(['abcd', '3456', 'EFGH'])
+   s.str.isalpha()
+   s.str.find('ab')
+
+- ``StringMethods.pad()`` and ``center()`` now accept ``fillchar`` option to specify filling character (:issue:`9352`)
```
Comment (jorisvandenbossche) [inline doc/source/whatsnew/v0.16.0.txt]: Can you specify `Series.str.pad()` instead of `StringMethods` (we now try to specify it as this in the methods)

Comment (jorisvandenbossche) [inline doc/source/whatsnew/v0.16.0.txt]: even `` :func:`Series.str.pad` `` should work now, so maybe use that (then it becomes a link to the docstring page

File: doc/source/whatsnew/v0.16.0.txt [hunk 4]
```diff
@@ -120,6 +121,44 @@ from a ``scipy.sparse.coo_matrix``:
...
+
+.. ipython:: python
+
+   s = Series(['12', '300', '25'])
+   s.str.pad(5, fillchar='_')
+
+
+- ``Series.str.slice_replace()``, which previously raised ``NotImplementedError`` (:issue:`8888`)
```
Comment (jorisvandenbossche) [inline doc/source/whatsnew/v0.16.0.txt]: add "Added" at the start of the sentence?

File: doc/source/whatsnew/v0.16.0.txt (hunk 3 above)
Comment (sinhrks) [inline doc/source/whatsnew/v0.16.0.txt]: I think directive should be `:meth:`. But neither `meth` and `func` are linked to docstring.

Comment (jorisvandenbossche) [inline doc/source/whatsnew/v0.16.0.txt]: After you build the documentation? Hmm, then I should look into it, as this should work now. For the meth/func, I actually think it doesn't matter, but not sure (in any case, meth is more appropriate here)

**Comment by jorisvandenbossche** [comment]
@sinhrks Merged this, thanks! I will look into the docstring linking thing. Ping #9599

**Pull request merged**